logos/src/logos/main.py

```
@@ -0,0 +1,280 @@
+        proxy_headers, forward_url, model_id, model_name, provider_id = out
+    with DBManager() as db:
+        if usage_id is not None:
+            db.set_forward_timestamp(usage_id)
+    # Forward Request
+    # Try multiple requesting methods. Start with streaming
+    try:
+        if "stream" not in json_data or json_data["stream"]:
+            print("Sending Streaming Request")
+            json_data["stream"] = True
+            return get_streaming_response(forward_url, proxy_headers, json_data, usage_id, provider_id, model_id)
+    except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)